src/main/java/seedu/address/model/util/Description.java

```diff
@@ -30,6 +30,13 @@ public Description(String description) {
         value = description;
     }
 
+    /**
+     * Constructs an empty description.
+     */
+    public static Description createEmptyDescription() {
+        return new Description("NIL");
```

Comment:
Not important now, but it would be good for all of modes to standardize using Optional.ofEmpty.